### ISSUE TITLE
Add sitting animation to lrfurn

### ISSUE THREAD

**Issue by amadin** [minetest-mods/homedecor_modpack#303]
Do you can add sitting animation to lrfurn mod?

**Comment by sofar** [comment]
I'm not opposed, and I'll merge it, but someone has to take the time to write this patch.

**Comment by tchncs** [comment]
Not just Irfurn - homedecor:chair miss that also - and its confusing since it already contains:
```
      on_rightclick = function(pos, node, clicker)
          pos.y = pos.y+0 -- where do I put my ass ?
          homedecor.sit(pos, node, clicker)
      end
```
But it wont work. Seems like its only working for armchairs - without animation - somehow. :/

**Comment by sofar** [comment]
I've been meaning to work with @everamzah to make this use the `cozy` mod and have that expose an easy and solid API to do this type of stuff.

**Comment by everamzah** [comment]
I actually recently tried to sit in a chair in homedecor.  Then I tried adding cozy and realized that the chairs have a square collision box, so you end up sitting in the air.

I'd like to try and experiment with making a table where it keeps track of attached players there, but haven't yet.  Anyway I have one concern in that there's a table in the beds mod that keeps track of laying players, so maybe use that?
